cmd/copy.go

```diff
@@ -638,6 +640,11 @@ func (raw rawCopyCmdArgs) cook() (CookedCopyCmdArgs, error) {
 		cooked.preserveSMBInfo = false
 	}
 
+	cooked.preservePOSIXProperties = raw.preservePOSIXProperties
+	if cooked.preservePOSIXProperties && !areBothLocationsPOSIXAware(cooked.FromTo) {
+		return cooked, fmt.Errorf("in order to use --preserve-posix-properties, both the source and destination must be POSIX-aware (Linux->Blob, Blob->Linux, Blob->Blob)")
```

Comment:
Nit nit: could just be me but trying to understand what you are trying to convey in the parantheses -- is it the source,destination pairing? Could we a bit more explicit say : "Supported POSIX-aware pairings are .." or even a ("vis" prefix in there)?